src/app/terms/page.js

```diff
@@ -0,0 +1,94 @@
+"use client";
+/* eslint-disable react/no-unescaped-entities */
+
+import { Box, Typography } from '@mui/material';
+
+
+
+export default function TermsAndConditionsPage() {
+  return (
+    <Box sx={{
+      fontFamily: 'Manrope, sans-serif',
+      backgroundColor: '#F2F2F2',
+      p: '40px 20px',
+      minHeight: '100vh',
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      }}>
+      <Box sx={{
+        width: '1117.73px',
+        height: '413.25px',
+        backgroundColor: '#E9ECEF',
+        borderRadius: '8.65849px',
+        p: '20px',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        marginTop: '100px', // Adjust this value to reduce the space
+        }}>
+        <Typography variant="h4" sx={{ fontWeight: 'bold' }}>
+          Terms and conditions
+        </Typography>
+        <Typography variant="body1">
+          Last updated: 21 September 2023
+        </Typography>
+      </Box>
+      <Box sx={{ width: 1116.16, p: '20px',fontSize: '16px', lineHeight: '24px' }}>
```

Comment:
use predefined font styles from theme. Otherwise it is unsupportable and impossible to change some font site-wide